data/random-teams.ts

```diff
@@ -733,6 +733,9 @@ export class RandomTeams {
 					if ((hasMove['substitute'] && !hasType['Fighting']) || hasMove['toxic'] && movePool.includes('substitute')) rejected = true;
 					if (moveid === 'closecombat' && (hasMove['highjumpkick'] || movePool.includes('highjumpkick')) && !counter.setupType) rejected = true;
 					break;
+				case 'coaching':
+					if (counter.setupType) rejected = true;
+					break;
```

Comment:
There's no need to make an ew case for this. Just add coaching on line 723.

Reply:
u rite